Modules/Registration/PDEDeformable/include/itkESMDemonsRegistrationFunction.hxx

```diff
@@ -171,7 +171,7 @@ ESMDemonsRegistrationFunction<TFixedImage, TMovingImage, TDisplacementField>::Co
   const FloatOffsetType &  itkNotUsed(offset)) -> PixelType
 {
   auto *    globalData = (GlobalDataStruct *)gd;
-  PixelType update;
+  PixelType update{};
```

Comment:
This added `{}` makes three `update.Fill(0.0)` calls later on in the same function unnecessary! No show-stopper, just something I noticed.

Reply:
Not a show stopper, but now is the time to remove those unnecessary update.Fill(0.0) commands.

Reply:
Thanks for the reviews. Done in ccb0065. Force pushed.